iModelCore/ECPresentation/Source/Content/ContentItemBuilder.cpp

```diff
@@ -149,13 +154,22 @@ rapidjson::Document ContentValuesFormatter::GetFormattedValue(ECPropertyCR prop,
         return GetFormattedStructValue(value, allocator);
     DIAGNOSTICS_HANDLE_FAILURE(DiagnosticsCategory::Content, "Unexpected ECProperty type");
     }
+
+/*---------------------------------------------------------------------------------**//**
+// @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+rapidjson::Document ContentValuesFormatter::GetFormattedValue(PrimitiveType type, Utf8StringCR extendedType, IECSqlValue const& value, rapidjson::MemoryPoolAllocator<>* allocator) const
+    {
+    return GetFallbackPrimitiveValue(type, extendedType, value, allocator);
+    }

```

Comment:
I don't understand this. It's an overload of the other `GetFormatterValue` function, so I'd expect the behavior to be similar except for different arguments, but it's not. To me it looks like this method is not needed at all, since we should call `GetFormattedPrimitiveValue` instead.

Reply:
Removed `GetFormattedValue`

Reply:
I see you changed the call to `GetFallbackPrimitiveValue`, but is that correct, though? I believe we should be calling `GetFormattedPrimitiveValue` instead.

Reply:
`GetFormattedPrimitiveValue` could not be used, because it requires `ECPropertyCR` parameter. This AddValue method is only used for calculated properties, renamed it to: `AddCalculatedPropertyValue`.